src/goodbase/goodbase_cli.py

```diff
@@ -394,6 +398,19 @@ def main(
     evg_config_file = os.path.expanduser(evg_config_file)
     evg_api = RetryingEvergreenApi.get_api(config_file=evg_config_file)
 
+    if branch:
+        if git_operation == GitAction.NONE:
+            git_operation = GitAction.CHECKOUT
+            LOGGER.debug(
+                "Branch name received and no git-operation specified, setting git-operation to checkout",
+                git_operation=git_operation,
+                branch=branch,
+            )
+        elif git_operation != GitAction.CHECKOUT:
+            raise ValueError(
+                f"-b or --branch option can only be used with CHECKOUT git-operation, but {git_operation} was given"
+            )
```

Comment:
The error message here looks good but can we use the same error messaging as below? This should give us some better formatting for the error message

```
            click.echo(
                click.style("Export file needs to be specified with `--export-file`", fg="red")
            )
            sys.exit(1)
```